src/components/shared/passphraseBackup/index.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import PassphraseBackup from '.';
+import { generatePassphrase } from '../../../utils/passphrase';
+
+describe('PassphraseBackup', () => {
+  let wrapper;
+  const props = {
+    account: {
+      passphrase: generatePassphrase(),
+    },
+    t: jest.fn(),
+  };
+
+  it('should show tip after copying passphrase', () => {
+    wrapper = mount(<PassphraseBackup {...props} />);
+    wrapper.find('CopyToClipboard').at(0).simulate('click');
+    expect('tip').toBeTruthy();
```

Comment:
💅 This assertion will be always true, no matter what you'll do with the component. It's not checking what's in `wrapper`

Reply:
I made the changes to check the text of the tip but somehow even when `this.state.showTip` is true, the test returns the content of the `tip` as empty. Can you check if there is something missing on the test?

Reply:
This was the problem: https://github.com/LiskHQ/lisk-hub/pull/2575/commits/8f2431b286c478a392acd6961aa20ddf2c1eda35

I also did two more commits on this file. See commit messages for more info...